README.md

```diff
@@ -78,6 +78,9 @@ The configuration supported by the plugin option includes:
     **Note:** Idiomatically the name is last element of the path but it need not be.
     For instance, the last element of the path might be the package's version, and the package would benefit
     from a more descriptive name.
+
+  * `module`: prefix to be stripped from the `go-gapic-package` used in the generated filenames.
```

Comment:
Is this `modulePrefix` or just `module`?

Reply:
actually prefix, i can provide something like this in case of `protoc-gen-go`:

```
		--go_out=new_folder \
		--go_opt=module=${DELOREAN_GENPROTO_GO_MODULE}/old_folder \
```

Reply:
The command line option is named `module`, as @valery-barysok indicates, I wanted to align with what already exists in other plugins. I named the containing variable in the code `modulePrefix` because I felt it was more readable, and to no get confused with the `go-gapic-package` value. I can change the variable to just `module` if you think that's better, but I think the cli option should stay as `module`.